apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/InnholdTilRedigerbartBrevHenter.kt

```diff
@@ -52,7 +52,7 @@ class InnholdTilRedigerbartBrevHenter(
 
         val kode = brevKodeMapping(brevkodeRequest)
         val tittel =
-            kode.tittel ?: (
+            kode.ferdigstilling.tittel ?: (
```

Comment:
At brevkoder har `redigering` og `ferdigstilling` var nytt for meg, og ikke påfallende innlysende _hva_ det er/gjør. Ser det heller ikke står noe om det i `Brevkoder`. 

Er det for å skille mal som brukes under redigering vs ferdigstilling? Fint om det står litt om tanken bak dette sånn at man slipper å saumfare koden for å prøve å forstå tanken bak det.

Reply:
Godt poeng, kan kommentere litt på det i koden. Men ja, det stemmer, det er nettopp for å ha det skiljet.